css/layout.css

```diff
@@ -0,0 +1,42 @@
+.hidden {
+  display: none !important;
+}
+html, body {
+  min-height: 100%;
+}
+
+body, main {
+  height:100vh;
+}
```

Comment:
Why? Typically, it is bad news to set the height of the `body`, it should be the size of its content